repository/src/main/resources/db/changelog/db.changelog-rdm-4351.xml

```diff
@@ -0,0 +1,18 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
we need to add this new field type to the ElasticSearch mappings too. Look at:
application.yml
After you do that, can you make sure you are able to import a spreadsheet with ElasticSearch turned on? If you try that at the moment, you will see
the import fails because on an ElasticSearch error. We need to fix that

Reply:
Added them